airbyte-server/src/main/java/io/airbyte/server/apis/AttemptApiImpl.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.server.apis;
+
+import io.airbyte.api.generated.AttemptApi;
+import io.airbyte.api.model.generated.InternalOperationResult;
+import io.airbyte.api.model.generated.SetWorkflowInAttemptRequestBody;
+import io.airbyte.persistence.job.JobPersistence;
+import io.airbyte.server.handlers.AttemptHandler;
+import javax.ws.rs.Path;
+
+@Path("/v1/attempt/set_workflow_in_attempt")
+public class AttemptApiImpl implements AttemptApi {
```

Comment:
Nit:  My feeling is that we should name these with the `Controller` suffix to match Micronaut nomenclature.  However, I don't feel strongly about this and if we have an internal reason to use `Impl` instead of `Controller`, I'm not going to push too hard on this.

Reply:
That's a good idea, I wasn't sure about the naming TBD, we don't have a `Impl` anywhere in our codebase.